seqr/views/utils/json_to_orm_utils.py

```diff
@@ -69,6 +70,9 @@ def update_model_from_json(model_obj, json, user=None, allow_unknown_keys=False,
         if getattr(model_obj, orm_key) != value:
             if orm_key in internal_fields and not (user and user.is_staff):
                 raise PermissionDenied('User {0} is not authorized to edit the internal field {1}'.format(user, orm_key))
+            has_updates = True
             setattr(model_obj, orm_key, value)
 
-    model_obj.save()
+    if has_updates:
+        model_obj.save()
```

Comment:
This saves a lot of time by only updating records if something has changed. There is never a reason we would want the old behavior of making unnecessary saves